### PR TITLE
[test-telemetry] add check for docker image

### PR DESCRIPTION
**Pull request by pra-moh**
### Description of PR
Adding a check for telemetry docker presence for each telemetry test

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Currently test run even though telemetry docker is not present which results in failure.

### CODE REVIEW

File: tests/telemetry/test_telemetry.py [hunk 1]
```diff
@@ -41,62 +41,84 @@ def setup_telemetry_forpyclient(duthost):
...
+    return False
+
 # Test functions
 def test_config_db_parameters(duthost):
     """Verifies required telemetry parameters from config_db.
     """
-    gnmi = duthost.shell('sonic-db-cli CONFIG_DB HGETALL "TELEMETRY|gnmi"', module_ignore_errors=False)['stdout_lines']
-    pytest_assert(gnmi is not None, "TELEMETRY|gnmi does not exist in config_db")
+    docker_present = verify_telemetry_dockerimage(duthost)
+    if docker_present:
```
Comment (lguohan) [inline tests/telemetry/test_telemetry.py:60]: the coding can be simplified by 
```
 if not docker_present:
    pytest.skip()
    return
```

File: tests/telemetry/test_telemetry.py [hunk 2]
```diff
@@ -64,11 +80,16 @@ def test_config_db_parameters(duthost):
             pytest_assert(str(value) == server_key_expected, "'server_key' value is not '{}'".format(server_key_expected))
         if str(key) == "server_crt":
             server_crt_expected = "/etc/sonic/telemetry/streamingtelemetryserver.cer"
-            pytest_assert(str(value) == server_crt_expected, "'server_crt' value is not '{}'".format(server_crt_expected))
+            pytest_assert(str(value) == server_crt_expected, "'server_crt' value is not '{}'".format(server_crt_expecteid))
```
Comment (jleveque) [inline tests/telemetry/test_telemetry.py:83]: Typo?

**Comment by hui-ma** [comment]
retest this please

**Comment by pra-moh** [comment]
retest this please

**Comment by pra-moh** [comment]
retest this please

**Pull request merged**